.travis.yml

```diff
@@ -12,6 +12,8 @@ addons:
   # this is working out-of-the-box on Travis, so we'll go with it in
   # CI for now.
   postgresql: "9.2"
+env:
+- RAILS_ENV=test
```

Comment:
https://github.com/ShelterTechSF/askdarcel-api/blob/master/config/database.yml#L58

Trying to get travis to run in this DB context. And not use the db hostname we set up for the development environment. (why the travis build is failing), looks like it is still picking up the hostname that is really only useful in docker.